doc/Roadmap.md

```diff
@@ -7,18 +7,19 @@ Please feel free to comment!
 * Push notifications:
     * [XEP-0357: Push Notifications](https://xmpp.org/extensions/xep-0357.html)
     * Amazon AWS SNS (Simple Notification Service)
-* Instant Stream Resumption
 * Full text search in archive
+* Experimental pipelining
+* Instant Stream Resumption
```

Comment:
The pipelining we have doesn't equal `Instant Stream Resumption` as in the proto-XEP.